crago.config.ts

```diff
@@ -0,0 +1,8 @@
+// craco.config.js
+module.exports = {
```

Comment:
is this file craco or crago? also not familiar with this

Reply:
Seems like it was a bridge with a breaking React 17 change. removed as there is a workaround.